sixtyfps_runtime/corelib/sharedvector.rs

```diff
@@ -65,9 +65,6 @@ fn alloc_with_capacity<T>(capacity: usize) -> NonNull<SharedVectorInner<T>> {
 /// Return a new capacity suitable for this vector
 /// Loosely based on alloc::raw_vec::RawVec::grow_amortized.
 fn capacity_for_grow(current_cap: usize, required_cap: usize, elem_size: usize) -> usize {
-    if current_cap >= elem_size {
```

Comment:
maybe this was meant to be required_cap.

So we do not re-allocates when it it already big enough.

But I guess the caller of capacity_for_grow already takes care of that?

Reply:
You're right, that makes more sense and the call site `push()` doesn't check, so that would have been a regression!